ecs-agent/tmds/handlers/fault/v1/handlers/handlers_test.go

```diff
@@ -902,49 +911,8 @@ func TestCheckNetworkBlackHolePort(t *testing.T) {
 	testNetworkFaultInjectionCommon(t, tcs, NetworkFaultPath(types.BlackHolePortFaultType, types.CheckNetworkFaultPostfix))
 }
 
-func generateNetworkLatencyTestCases(name, expectedHappyResponseBody string) []networkFaultInjectionTestCase {
-	happyNetworkLatencyReqBody := map[string]interface{}{
-		"DelayMilliseconds":  delayMilliseconds,
-		"JitterMilliseconds": jitterMilliseconds,
-		"Sources":            ipSources,
-	}
+func generateCommonNetworkLatencyTestCases(name string) []networkFaultInjectionTestCase {
 	tcs := []networkFaultInjectionTestCase{
-		{
-			name:                 fmt.Sprintf("%s success", name),
-			expectedStatusCode:   200,
-			requestBody:          happyNetworkLatencyReqBody,
-			expectedResponseBody: types.NewNetworkFaultInjectionSuccessResponse(expectedHappyResponseBody),
-			setAgentStateExpectations: func(agentState *mock_state.MockAgentState) {
-				agentState.EXPECT().GetTaskMetadata(endpointId).
-					Return(happyTaskResponse, nil).
-					Times(1)
-			},
-		},
-		{
-			name:               fmt.Sprintf("%s unknown request body", name),
-			expectedStatusCode: 200,
-			requestBody: map[string]interface{}{
-				"DelayMilliseconds":  delayMilliseconds,
-				"JitterMilliseconds": jitterMilliseconds,
-				"Sources":            ipSources,
-				"Unknown":            "",
-			},
-			expectedResponseBody: types.NewNetworkFaultInjectionSuccessResponse(expectedHappyResponseBody),
-			setAgentStateExpectations: func(agentState *mock_state.MockAgentState) {
-				agentState.EXPECT().GetTaskMetadata(endpointId).
-					Return(happyTaskResponse, nil).
-					Times(1)
-			},
-		},
-		{
-			name:                 fmt.Sprintf("%s no request body", name),
```

Comment:
Why do we remove this test case?

Reply:
Got it. Thanks for the catch! Will add this back in the next PR.